packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -512,8 +491,8 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/req
      condition: grafana.enabled
 +
 +  - name: prometheus-adapter
```

Comment:
`rancher-pushprox` also needs a `version` bump on the `Chart.yaml` as well as version bumps throughout the dependencies file since it is also maintained in `rancher/charts` as a separate chart. (Note: It probably didn't affect you when testing due to the way it is packaged via local chart archives)

Also, @bashofmann I was planning to make a separate PR updating the repositories in https://github.com/rancher/charts/pull/846 but would it be possible for you to just update this PR with the changes? `rancher/charts` doesn't work nicely with multiple people making PRs to update the same file in a patch, so that would be easier than performing a rebase after one or the other is merged in.

Reply:
I bumped the version of `ranher-pushprox` in all places I could find and cherry-picked your commit from #846 into this branch.